docs/usage/self-hosted-configuration.md

```diff
@@ -328,6 +328,14 @@ For example, if you set `dockerChildPrefix=myprefix_` then the final container c
 !!! note
     Dangling containers are only removed when Renovate runs again with the same prefix.
 
+## dockerCliOptions
+
+Adds command-line docker CLI options to Renovate sidecar Docker containers.
+
+For example, `--memory=1g --cpus="1.5"`.
```

Comment:
I'd like a real config option example here, something similar to:

```json
{
  "dockerCliOptions": ["this-is-a-bad-config-do-not-copy"]
}
```

Reply:
It's not completely bad or good

Reply:
🤔  think those are already valid options.